### ISSUE TITLE
uBlock doesn't work! (? blocked out of total ?)

### ISSUE THREAD

**Issue by AIndoria** [uBlock-LLC/uBlock#255]
Shows me the "? requests blocked on this page. ? total."

Also the options page has nothing but checkboxes on top. That's about it. If you refresh it, the options appear but with "Page not responding." error. (My mouse pointer turns to Windows "busy" mode.) Help? :/

Edit : I uninstalled chrome, even sys restored to three days ago. Nope.

**Comment by gorhill** [comment]
What browser version?

**Comment by AIndoria** [comment]
39.0.2166.2 Dev. I went to dev just today after it didn't work on Beta.

**Comment by gorhill** [comment]
Enable _"Predict network actions to improve page load performance"_.

Confirm if this is a duplicate of https://github.com/gorhill/uBlock/issues/247.

**Comment by AIndoria** [comment]
Sorry, It was enabled. I disabled it, and re-enabled it. Same issue.

Additionally, the "page not responding" error is for "Third party filters" page.

**Comment by AIndoria** [comment]
Okay, it seems to fix itself as soon as I disable Hardware acceleration but appears again when it's enabled.

**Comment by gorhill** [comment]
Work fine here using uBlock 0.6.4.0 with Chrome 39.0.2166.2 dev-m. The "? blocked out of total ?" symptom is what would happen if uBlock's javascript is prevented from executing. In general, I can't promise the extension will work properly on non-stable version of Chromium. Did you try in a stable version?

**Comment by AIndoria** [comment]
See my above reply :) And no, I've just been using Beta forever and it's worked fine on that. I'll report if something else pops up but Disabling hardware accel. seems to work for now.

**Comment by gorhill** [comment]
> Okay, it seems to fix itself as soon as I disable Hardware acceleration but appears again when it's enabled

Interesting finding. You may want to add this finding to this bug: https://code.google.com/p/chromium/issues/detail?id=415071

**Comment by AIndoria** [comment]
I will, thanks.

**Comment by gorhill** [comment]
I am going to assume it is a dup of #247
